packages/BlockQuote/package.json

```diff
@@ -0,0 +1,37 @@
+{
+  "name": "@tds/community-block-quote",
+  "version": "1.0.0-preview",
+  "description": "",
+  "main": "index.cjs.js",
+  "module": "index.es.js",
+  "license": "MIT",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/telus/tds-community.git"
+  },
+  "publishConfig": {
+    "access": "public"
+  },
+  "author": "TELUS digital",
+  "engines": {
+    "node": ">=8"
+  },
+  "bugs": {
+    "url": "https://github.com/telus/tds-community/issues"
+  },
+  "homepage": "http://tds.telus.com",
+  "peerDependencies": {
+    "react": "^16.8.2",
+    "react-dom": "^16.8.2",
+    "styled-components": "^4.2.0"
```

Comment:
Double check to see if a feature in 4.2.0 is needed, otherwise this should be 4.1.3 to align with the rest of our components